src/urllib3/__init__.py

```diff
@@ -20,6 +20,19 @@
 from .util.retry import Retry
 from .util.timeout import Timeout
 
+try:
+    import urllib3_secure_extra  # type: ignore # noqa: F401
+except ImportError:
```

Comment:
I think in 3.7+?

```suggestion
except ModuleNotFoundError:
```

Reply:
According to Python [doc](https://docs.python.org/3/library/exceptions.html#ModuleNotFoundError), `ModuleNotFoundError` is a subclass of `ImportError`.  Ya I agree with you,  `ModuleNotFoundError` seems more intuitive and serves right purpose here though `except ImportError` still able to catch the exception.

Reply:
The purpose of ModuleNotFoundError is to distinguish cases where something breaks in a significant way from the module simply not existing.

If an ImportError that's not a ModuleNotFoundError shows up here then users will want to know about it